asreview/webapp/utils/paths.py

```diff
@@ -55,6 +55,18 @@ def get_project_file_path(project_id):
     return Path(get_project_path(project_id), "project.json")
 
 
+def get_tmp_path(project_id):
```

Comment:
Perhaps we can make use of tempfile? https://docs.python.org/3.8/library/tempfile.html (tempfile.mkstemp)

Reply:
Good suggestion, I will implement this. Maybe this is a good way to export the CSV as well. Easier to trace than exporting a buffer.